docs/whats-new.md

```diff
@@ -6,14 +6,63 @@ description: Summary of the latest updates to this documentation.
 # What's new in the documentation?
 
 The latest major MetaMask documentation updates are listed by the month they were added to the documentation. 
-For a comprehensive list of recent product changes, visit the "Release Notes" section at the bottom of the [MetaMask developer page](https://metamask.io/developer/).
+For a comprehensive list of recent product changes, visit the "Release Notes" section at the bottom
+of the [MetaMask developer page](https://metamask.io/developer/).
+
+## March 2024
+
+- Documented [how to restrict a Snap's RPC API methods](https://github.com/MetaMask/metamask-docs/pull/1223).
+  ([#1223](https://github.com/MetaMask/metamask-docs/pull/1223))
+- Documented [Snap-defined timeouts](/snaps/reference/permissions/#snap-defined-timeouts).
+  ([#1224](https://github.com/MetaMask/metamask-docs/pull/1224) and
+  [#1227](https://github.com/MetaMask/metamask-docs/pull/1227))
+- Restructured Wallet documentation to emphasize [connecting using EIP-6963](/wallet/how-to/connect).
+  ([#1214](https://github.com/MetaMask/metamask-docs/pull/1214))
+- Documented [Snaps custom name resolution](/snaps/features/custom-name-resolution).
+  ([#924](https://github.com/MetaMask/metamask-docs/pull/924))
+- Deprecated `snaps-types` and `snaps-ui` packages and updated
+  [Snaps tutorials](/snaps/learn/tutorials) to match latest tooling.
+  ([#1206](https://github.com/MetaMask/metamask-docs/pull/1206) and
+  [#1210](https://github.com/MetaMask/metamask-docs/pull/1210))
+- Documented new [Snaps custom UI](/snaps/features/custom-ui) components and improved examples.
+  ([#1209](https://github.com/MetaMask/metamask-docs/pull/1209) and
+  [#1225](https://github.com/MetaMask/metamask-docs/pull/1225))
+- Removed "Flask" label from [Snaps home pages](/snaps/reference/entry-points/#onhomepage) and
+  [`snap_getLocale`](/snaps/reference/snaps-api/#snap_getlocale).
+  ([#1204](https://github.com/MetaMask/metamask-docs/pull/1204))
+- Restructured [SDK documentation](/wallet/how-to/use-sdk) to de-emphasize it as a recommended method.
+  ([#1200](https://github.com/MetaMask/metamask-docs/pull/1200) and
+  [#1230](https://github.com/MetaMask/metamask-docs/pull/1230))
+- Deprecated the following [provider API](/wallet/reference/provider-api) elements:
+  - `chainId` property
+  - `networkVersion` property
+  - `selectedAddress` property
+  - `networkChanged` event
+  - `net_version` support in `send()`
+- Added [Snaps best practices section](/snaps/learn/best-practices).
+  ([#1187](https://github.com/MetaMask/metamask-docs/pull/1187))
+- Updated [Snaps CLI reference](/snaps/reference/cli) with Webpack configuration options.
+  ([#1178](https://github.com/MetaMask/metamask-docs/pull/1178))
+- Documented [how to use Wagmi with the SDK](/wallet/how-to/use-sdk/3rd-party-libraries/wagmi).
```

Comment:
I think Eric Bershard might have mentioned this does not work as described. I don't know if we want to highlight this until it is corrected?